### PR TITLE
Fixed prsysmgmt issue with connection via JMX and updated docs

### PR DESCRIPTION
**Pull request by petehayes**
I fixed the issue related to JMX and updated the docs a bit.

### CODE REVIEW

File: docker-entrypoint.sh [hunk 1]
```diff
@@ -1,6 +1,10 @@
 #!/bin/bash
 set -x
 
+# Define the JDBC_URL variable based on inputs
+JDBC_URL="jdbc:postgresql://${DB_HOST}:${DB_PORT}/${DB_NAME}${JDBC_URL_SUFFIX}"
```
Comment (petehayes) [inline docker-entrypoint.sh]: shoot, just noticed that I hardwired postgresql here.  I'll need to fix that.

File: Dockerfile [hunk 2]
```diff
@@ -12,8 +12,9 @@ ENV DB_PORT 5432
 ENV DB_NAME pega
 
 # Provide variables for the JDBC connection string
-ENV JDBC_URL "jdbc:postgresql://${DB_HOST}:${DB_PORT}/${DB_NAME}"
 ENV JDBC_CLASS org.postgresql.Driver
+ENV JDBC_DB_TYPE postgresql
```
Comment (cdancy) [inline Dockerfile]: If not hard-coding postgres URL does it make sense to hard-code jdbc_class and db_type?

Comment (cdancy) [inline Dockerfile]: Maybe remove `JDBC_DB_TYPE` as well and just parse it from the passed `JDBC_URL`? That's what we do here in our `gradle-prpc-plugin` and would remove one more variable to track.

**Comment by petehayes** [comment]
This one doesn't require the user to pass in the jdbc_url - it is
constructed in the docker-entrypoint script.  These are just the defaults
which do default to postgres.

**Comment by petehayes** [comment]
I made those changes.  Thanks!

**Comment by browm2** [comment]
Any time!

**Pull request merged**